### PR TITLE
Add onnx export for floor, ceil, log2 and prim::shape

### PR DESCRIPTION
**Pull request by BowenBao**

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
         dim = _parse_arg(dim, 'i')
         keepdim = _parse_arg(keepdim, 'i')
         return g.op('ArgMin', input, axis_i=dim, keepdims_i=keepdim)
+
+
+@parse_args('v')
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: i think no need to add `parse_args` in these cases (i.e., all inputs are tensors): https://github.com/pytorch/pytorch/blob/c484cf43a02863efd2f4a76aad43246fb0191ab5/torch/onnx/symbolic.py#L114

File: torch/onnx/symbolic.py [hunk 2]
```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
...
+def floor(g, self):
+    return g.op('Floor', self)
+
+
+@parse_args('v')
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: ditto

File: torch/onnx/symbolic.py [hunk 3]
```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
...
+def ceil(g, self):
+    return g.op('Ceil', self)
+
+
+@parse_args('v')
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: ditto

File: torch/onnx/symbolic.py [hunk 4]
```diff
@@ -1710,3 +1710,24 @@ def argmin(g, input, dim, keepdim):
...
+def log2(g, self):
+    _ln2 = 0.693147180559945309
+    return g.op('Div', log(g, self), g.op('Constant', value_t=torch.Tensor([_ln2])))
+
+
+@parse_args('v')
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: can we also add a test case for this one?

**Review by houseroad — COMMENTED** [review]
Thanks for the PR. Please address inline comments.

File: torch/onnx/symbolic.py (hunk 1 above)
Comment (BowenBao) [reply torch/onnx/symbolic.py]: done

File: torch/onnx/symbolic.py (hunk 2 above)
Comment (BowenBao) [reply torch/onnx/symbolic.py]: done

File: torch/onnx/symbolic.py (hunk 3 above)
Comment (BowenBao) [reply torch/onnx/symbolic.py]: done

File: torch/onnx/symbolic.py (hunk 4 above)
Comment (BowenBao) [reply torch/onnx/symbolic.py]: done.

File: torch/onnx/symbolic.py [hunk 5]
```diff
@@ -1726,3 +1726,12 @@ def argmin(g, input, dim, keepdim):
...
         keepdim = _parse_arg(keepdim, 'i')
         return g.op('ArgMin', input, axis_i=dim, keepdims_i=keepdim)
+
+
+def log2(g, self):
+    _ln2 = 0.693147180559945309
+    return g.op('Div', log(g, self), g.op('Constant', value_t=torch.Tensor([_ln2])))
+
+
+def prim_shape(g, self):
```
Comment (houseroad) [inline torch/onnx/symbolic.py]: Interesting, I am wondering why this would work with prim::shape node.

Comment (houseroad) [reply torch/onnx/symbolic.py]: I need to think about whether this is a good way to handle prim::nodes

**Review by houseroad — APPROVED** [review]
Looks good.

Once we add support for custom domain op, we should refactor prim domain op.

**Pull request closed without merging**